independent-projects/enforcer-rules/src/main/resources/enforcer-rules/default-quarkus.xml

```diff
@@ -0,0 +1,123 @@
+<enforcer>
+    <rules>
+        <dependencyConvergence/>
+        <requireJavaVersion>
+            <version>[${maven.compiler.release},)</version>
```

Comment:
Same for this one. I think I would refrain from adding it here.

Reply:
I am not 100% sure about this to be honest. If not here, I'd expect that at least this rule would need to be included in each project, which may lead to values that are not expected in  Quarkus (eg. Extensions being compiled/released using Java 17+ instead of 11)

Reply:
Same as above: I'd be interested to get `<supported-maven-versions>` in Camel Quarkus.